config/src/test/java/com/alibaba/nacos/config/server/utils/GroupKey2Test.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright 1999-2018 Alibaba Group Holding Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.alibaba.nacos.config.server.utils;
+
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
+import org.springframework.test.context.web.WebAppConfiguration;
+
+@RunWith(SpringJUnit4ClassRunner.class)
+@WebAppConfiguration
+public class GroupKey2Test {
+
+    @Test
+    public void test_parseGroupKey2_非法的() {
+        String key = "11111+222+333333+444";
+        try {
+            GroupKey2.parseKey(key);
+            Assert.fail();
+        } catch (IllegalArgumentException e) {
+            System.out.println(e.toString());
```

Comment:
If this is their existing tests that you've moved, i'd keep their original test file intact & have ours in a different file instead.

Reply:
Just put them in the same file since KeyGroup/KeyGroup2 are nearly exactly the same.